Added interface and logic with mock data

shooin13 opened the pull request:
- Added interface 
- Added logic using mock data
- Removed landscape orientation

shooin13 committed:
Added interface and logic with mock data

DanilaG commented on MovieQuiz/Presentation/MovieQuizViewController.swift:
```diff
@@ -1,72 +1,165 @@
+  private let questions: [QuizQestion] = [
+    QuizQestion(
+      image: "The Godfather",
+      text: "Рейтинг этого фильма больше чем 6",
```
Кажется тут потерялся знак вопроса в конце)

shooin13 replied:
Все пофиксил)

shooin13 committed:
- Status bar changed to lightContent only for MovieQuizViewController
- Removed ability to press buttons during the correct/incorrect result is shown
- Removed «Models» and «Services» empty folders
- Added missing question mark in mock date (first question)
- Changed case of text in buttons (uppercased->first letter only) by changing text
- Removed border around image after starting over
